stream-chat-android-ui-components/src/main/res/values/strings.xml

```diff
@@ -33,16 +33,19 @@
     <string name="stream_add_channel_back_icon_description">Back</string>
     <string name="stream_add_channel_add_member_icon_description">Add member</string>
     <string name="stream_add_channel_user_checked_description">User checked</string>
+    <string name="stream_add_channel_empty_users_description">No users</string>
     <string name="stream_add_channel_title">New Chat</string>
     <string name="stream_add_member_title">TO:</string>
     <string name="stream_add_member_hint">Type a name</string>
     <string name="stream_create_group">Create a Group</string>
     <string name="stream_add_channel_user_list_title">On the platform</string>
+    <string name="stream_add_channel_empty_users_title">No users matches these keywords…</string>
```

Comment:
Mistypo here should be `No user_`
![image](https://user-images.githubusercontent.com/11807573/100237737-6000a700-2f40-11eb-9a32-cbb644d95f8a.png)

Reply:
nice catch!